fbw-a380x/src/wasm/systems/a380_systems/src/air_conditioning/mod.rs

```diff
@@ -2075,19 +2486,20 @@ mod tests {
             );
         }
 
-        #[test]
-        fn cabin_vs_changes_to_takeoff() {
-            let test_bed = test_bed()
-                .set_on_ground()
-                .iterate(50)
-                .set_takeoff_power()
-                .iterate_with_delta(400, Duration::from_millis(10));
-
-            assert!(
-                (test_bed.cabin_vs() - Velocity::new::<foot_per_minute>(-400.)).abs()
-                    < Velocity::new::<foot_per_minute>(20.)
-            );
-        }
+        // Test fails because the packs generate more air but the outflow valve hasn't been adjusted for A380 yet
+        // To be uncommented when pressurization system is adapted to a380
+        // #[test]
+        // fn cabin_vs_changes_to_takeoff() {
+        //     let test_bed = test_bed()
+        //         .set_on_ground()
+        //         .iterate(50)
+        //         .set_takeoff_power()
+        //         .iterate_with_delta(400, Duration::from_millis(10));
+        //     assert!(
+        //         (test_bed.cabin_vs() - Velocity::new::<foot_per_minute>(-400.)).abs()
+        //             < Velocity::new::<foot_per_minute>(20.)
+        //     );
+        // }
```

Comment:
I think we usually let the test in but use  #[ignore]

Reply:
Thanks for the tip! I've marked all pressurization tests with the #[ignore] flag since the system is not yet modelled.